lib/chef/provisioning/aws_driver/driver.rb

```diff
@@ -1010,18 +1022,20 @@ def default_aws_keypair(action_handler, machine_spec)
       default_key_name
     end
 
-    def create_servers(action_handler, specs_and_options, parallelizer, &block)
+    def create_servers(action_handler, specs_and_options, parallelizer)
       specs_and_servers = instances_for(specs_and_options.keys)
 
       by_bootstrap_options = {}
       specs_and_options.each do |machine_spec, machine_options|
-        actual_instance = specs_and_servers[machine_spec]
-        if actual_instance
-          if actual_instance.status == :terminated
-            Chef::Log.warn "Machine #{machine_spec.name} (#{actual_instance.id}) is terminated.  Recreating ..."
+        instance = specs_and_servers[machine_spec]
+        if instance
```

Comment:
The `actual_instance` -> `instance` change makes it harder to see what's actually changed for the v2 driver.